packages/core/src/validation.ts

```diff
@@ -184,6 +185,9 @@ const initOptions = Joi.object({
   gas: Joi.object({
     get: Joi.function().required(),
     stream: Joi.function().required()
+  }),
+  connect: Joi.object({
+    showSidebar: Joi.boolean()
```

Comment:
```suggestion
  connect: connectOptions
```

Could we bring the `connectOptions` up a few lines and then reuse it here?

Reply:
Oh yeah duh! Thanks man!